packages/starknet-snap/src/rpcs/executeTxn.test.ts

```diff
@@ -90,41 +90,53 @@ const prepareMockExecuteTxn = async (
 describe('ExecuteTxn', () => {
   let callsExample: any;
 
-  it('executes transaction correctly if the account is deployed', async () => {
-    callsExample = callsExamples[0];
-    const {
-      account,
-      createAccountSpy,
-      executeTxnRespMock,
-      getEstimatedFeesSpy,
-      getEstimatedFeesRepsMock,
-      request,
-    } = await prepareMockExecuteTxn(
-      callsExample.hash,
-      callsExample.calls,
-      callsExample.details,
-      true,
-    );
+  it.each([
+    {
+      calls: callsExamples[0],
```

Comment:
nit:

IMO, the fixture actually do doing in a good format, 
as you have to using index to identify it is array or not array call

i will suggest change the fixture format, by giving some meaningful key on that (please refer to other fixture)